### PR TITLE
Minor changes to readme

### PR DESCRIPTION
**Pull request by dhruvbhagtani**
Very minor readme changes

### CODE REVIEW

**Review by navidcy — APPROVED** [review]
lgtm

**Comment by navidcy** [comment]
Similar typos exist probably in the installation instructions in the docs. Since we are at it, can we fix those also?

File: README.md [hunk 1]
```diff
@@ -6,28 +6,28 @@
...
-1. A cool idea for a new regional MOM6 domain
-2. A working MOM6 executable on a machine of your choice. 
-3. A bathymetry file that at least covers your domain
-4. 3D ocean forcing files *of any resolution* on your choice of A, B or C Arakawa grid
-5. Surface forcing files (eg ERA or JRA reanalysis)
+## What you need to get started:
+1. a cool idea for a new regional MOM6 domain,
+2. a working MOM6 executable on a machine of your choice, 
+3. a bathymetry file that at least covers your domain,
+4. 3D ocean forcing files *of any resolution* on your choice of A, B or C Arakawa grid,
```
Comment (navidcy) [inline README.md:19]:
```suggestion
4. 3D ocean forcing files *of any resolution* on your choice of A, B, or C Arakawa grid,
```

File: README.md [hunk 2]
```diff
@@ -6,28 +6,28 @@
...
+5. surface forcing files (eg ERA or JRA reanalysis), and
```
Comment (navidcy) [inline README.md:20]:
```suggestion
5. surface forcing files (e.g., from ERA or JRA reanalysis), and
```

File: README.md [hunk 3]
```diff
@@ -6,28 +6,28 @@
...
 6. [GFDL's FRE tools](https://github.com/NOAA-GFDL/FRE-NCtools) must be downloaded and compiled on the machine you are using.
```
Comment (navidcy) [inline README.md:21]:
```suggestion
6. [GFDL's FRE tools](https://github.com/NOAA-GFDL/FRE-NCtools) be downloaded and compiled on the machine you are using.
```

**Comment by dhruvbhagtani** [comment]
Yes, let's fix all typos!

**Comment by navidcy** [comment]
Could you have a look at https://github.com/dhruvbhagtani/regional-mom6/blob/db-readme-changes/docs/installation.md as well?

**Pull request merged**